docs/lib/graphqlapi/fragments/ios/getting-started/12_amplifyConfig.md

```diff
@@ -0,0 +1 @@
+Upon completion, `amplifyconfiguration.json` will be updated to reference provisioned backend storage resources.  Note that these files should already be a part of your project if you followed the [Project setup walkthrough](~/lib/project-setup/create-application.md).
```

Comment:
Not sure what 'storage resources' means?

Reply:
Agree, it seems wrong. Updated.